pipeline/post-results.groovy

```diff
@@ -66,6 +66,30 @@ node(nodeName) {
 
         //Remove the sync results folder
         sh script: "rclone purge ${remoteName}:${reportBucket}/${resultDir}"
+
+        // Update RH recipe file
+        def testStatus = msgMap["test"]["result"]
+        def composeInfo = readJSON text: msgMap["build"]
+
+        if ( testStatus == "SUCCESS" && composeInfo ){
+            def tierLevel = msgMap["pipeline"]["name"]
+            def rhcsVersion = sharedLib.getRHCSVersionFromArtifactsNvr()
+            majorVersion = rhcsVersion["major_version"]
+            minorVersion = rhcsVersion["minor_version"]
+
+            def latestContent = sharedLib.readFromReleaseFile(
+                    majorVersion, minorVersion
+                )
+
+            if ( latestContent.containsKey(tierLevel) ) {
+                latestContent[tierLevel] = composeInfo
```

Comment:
Shouldn't the `composeInfo` hold both RH build location information along with IBM-C location

Reply:
As discussed, not needed.